### PR TITLE
update few app entries

### PR DESCRIPTION
**Pull request by pnemade**
I would like to propose 
1) update to existing bodhi app information
2) replace releng-dash with PDC
3) add Mdapi
4) add Pagure with its icon

Please review and provide feedback if needed any changes to this PR. This PR will also address issues reported https://github.com/fedora-infra/apps.fp.o/issues/42 and https://github.com/fedora-infra/apps.fp.o/issues/43

I also found that with adding more apps it becomes more difficult for end users to read the information. I would like to propose increase in levelDistance to 200 to have slightly more readable view. If that is okay I can file new PR for it.

### CODE REVIEW

File: data/apps.yaml [hunk 1]
```diff
@@ -144,14 +144,15 @@ children:
...
+            url:    https://pdc.fedoraproject.org/
             description: >
-                Track the status of the Fedora Release Engineering process.
-                Did the latest rawhide get rsynced out?  How about for the
-                secondary arches?  This read-only dashboard can help you
-                make a quick check.
+                The Product Definition Center is repository and API for
+                storing and querying metadata related to packages, product
+                releases, engineering processes and artifacts which are
+                required to support automation of Red Hat engineering
```
Comment (pypingou) [inline data/apps.yaml]: Should we mention Red Hat engineering on a Fedora page?

Comment (pnemade) [inline data/apps.yaml]: Actually I just copied the description from https://pdc.fedoraproject.org page.  But, you are right I will update PR tomorrow.

**Comment by pnemade** [comment]
Updated PR, please review it again.

**Comment by pypingou** [comment]
Looks good to me, thanks! :)

**Pull request merged**